### ISSUE TITLE
multisession: accessibility doesn't work on ubuntu/windows

### ISSUE THREAD

**Issue by midleman** [posit-dev/positron#7726]
## System details:

Windows and Ubuntu only. Mac is fine.

## Steps to reproduce the issue:

1. Click session
2. Use arrow and enter keys to rename session
3. Verify user is able to rename session - FAIL

## Expected or desired behavior:

Any OS should have working accessibility keys.

**Comment by midleman** [comment]
### Verified Fixed
Positron Version(s) : [2025.06.0-145](https://github.com/posit-dev/positron-builds/releases/tag/2025.06.0-145)
OS Version(s) : MacOS

### Test scenario(s) 
Confirmed accessibility tests now run and pass on Windows/Ubuntu/Mac. Closing ticket.

### Link(s) to test cases run or created: 
https://github.com/posit-dev/positron/blob/276cdf2c4f4b7c3584cfb7e3375a3d9494057254/test/e2e/tests/sessions/session-mgmt.test.ts#L175-L200
